samples/client/petstore/typescript-angular-v4.3/npm/model/tag.ts

```diff
@@ -21,3 +21,5 @@ export interface Tag {
     name?: string;
 
 }
+
```

Comment:
And these empty lines?

Reply:
same as above

Reply:
I see, and did someone delete the newline at the end? And does your output contain one or two blank lines in the end? It appears to be two, but maybe that's just Github

Reply:
@sebastianhaas i cannot tell where the newlines come from, but they also appear in e.g. https://github.com/swagger-api/swagger-codegen/pull/6769, so it is just about re-generating the files